app.py

```diff
@@ -1,74 +1,127 @@
 #!/usr/bin/env python3
 from flask import *
 from requests import post
-from json import load, dump
```

Comment:
This breaks the program. Please readd this line, otherwise the block directly below it won't work.

Reply:
Strange I tested both the requirements and this one and worked...will add it back